types/networks/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 // Copyright 2019-2022 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
+#![feature(variant_count)]
```

Comment:
This seems like a shaky feature, not only we would be bolting ourselves to nightly, but also in future it may not fly https://github.com/rust-lang/rust/issues/73662#issuecomment-1183512441

If we want to compare enum lengths, maybe it would be "more stable" with https://docs.rs/strum/0.24.0/strum/derive.EnumCount.html